test-resources/clj-kondo.exports/marick/midje/marick/midje.clj

```diff
@@ -17,15 +27,50 @@
 
 (defn ^:private do-form [forms]
   (hooks/list-node
-    (concat [(hooks/token-node 'do)]
-            forms)))
+   (concat [(hooks/token-node 'do)]
+           forms)))
 
-(defn tabular [{:keys [node]}]
-  (let [[facts vec-bindings & bindings] (rest (:children node))]
+(defn ^:private table-variable? [node]
+  (let [sexpr (hooks/sexpr node)]
+    (and (symbol? sexpr)
+         (string/starts-with? (str sexpr) "?"))))
+
+(defn ^:private handle-fact-outside-tabular [children arrow]
+  (let [pos-arrow-index (inc (index-of arrow children))
+        body (do-form children)
+        bindings (->> children
+                      (map-indexed (fn [i el]
+                                     (when (> i pos-arrow-index)
+                                       el)))
+                      (remove nil?))]
```

Comment:
instead of `map-index` + `remove` could you do a `(drop pos-arrow-index children)`?

Reply:
Another thing you can do to not have to calculate `pos-arrow-index` is:
```clojure
(->> children
     (drop-while #(not (= arrow %)))
     rest)
```
not sure if it is clearer though. For me avoiding `keep-indexed` and `map-indexed` (which I had to lookup the docstrings for) helps readability

Reply:
it seems better indeed, I just needed to add a drop:
```clojure
(->> children
     (drop-while #(not (= arrow %)))
     rest
     (drop 1))
```